python/lsst/pipe/tasks/make_direct_warp.py

```diff
@@ -547,6 +620,28 @@ def process(
 
         return warped_exposure
 
+    @staticmethod
+    def _filterInputs(indices, inputs):
+        """Filter task inputs by their indices.
+
+        Parameters
+        ----------
+        indices : `list` [`int`]
+        inputs : `dict` [`list`]
+            A dictionary of input connections to be passed to run.
```

Comment:
It's not at all clear what these collections actually contain.

Reply:
Agree. I lifted these from `makeWarp` instead of importing from there, given the anticipated deprecation. I've improved the wording a bit but not too worried given that it's a private method as opposed to this being a public method of `MakeWarpTask`.